smarts/core/vehicle.py

```diff
@@ -312,6 +313,10 @@ def build_agent_vehicle(
         vehicle_type = "passenger"
         chassis_dims = VEHICLE_CONFIGS[vehicle_type].dimensions
 
+        if isinstance(mission_planner.mission.task, UTurn):
+            if mission_planner.mission.task.initial_speed:
+                initial_speed = mission_planner.mission.task.initial_speed
```

Comment:
OK, this is fine. A new issue needs to be brought up around the consideration of generalising this initial speed parameter by acquiring it from the mission itself rather than from the task.

Reply:
Thanks @Gamenot , I created #563 .